src/Uno.UI/UI/Xaml/Controls/Page/Page.mux.cs

```diff
@@ -35,7 +48,22 @@ private protected override void OnLoaded()
 
 					focusManager.InitialFocus = true;
 
-					TrySetFocusedElement(spFirstFocusableElementDO);
+					try
+					{
+						var focusUpdated = this.SetFocusedElement(
+							spFirstFocusableElementDO,
+							FocusState.Programmatic,
+							false);
+					}
+					catch (Exception ex)
+					{
+						if (this.Log().IsEnabled(LogLevel.Error))
+						{
+							this.Log().LogError($"Setting initial page focus failed: {ex}");
+						}
+					}
+
+					focusManager.InitialFocus = false;
```

Comment:
Keep the original update please (for performance) :)

Reply:
Sorry, wrong conflict resolution